fuzz/fuzz_targets/cranelift-fuzzgen.rs

```diff
@@ -56,7 +59,13 @@ fuzz_target!(|testcase: TestCase| {
         let mut env = FunctionStore::default();
         env.add(testcase.func.name.to_string(), &testcase.func);
 
-        let state = InterpreterState::default().with_function_store(env);
+        let state = InterpreterState::default()
+            .with_function_store(env)
+            .with_libcall(LibCall::IshlI64, &|args| match &args[..] {
+                [DataValue::I64(_), DataValue::I64(b)] if *b >= 63 => smallvec![DataValue::I64(0)],
+                [DataValue::I64(a), DataValue::I64(b)] => smallvec![DataValue::I64(a.shl(b))],
+                _ => unreachable!(),
+            });
```

Comment:
I saw some sources online that point to this being the correct behaviour, but nothing definitive. Does anyone know if this is correct?